client/my-sites/checkout/checkout-thank-you/domains/domain-mapping-thank-you.jsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+import { ThankYou } from 'calypso/components/thank-you';
+
+import { CALYPSO_CONTACT, SUPPORT_ROOT } from 'calypso/lib/url/support';
+import { Button } from '@automattic/components';
+import domainConnectedSuccess from 'calypso/assets/images/illustrations/domain-connected-success.svg';
+
+import './style.scss';
+
+const DomainMappingThankYou = ( { domain, translate } ) => {
+	return (
+		<ThankYou
+			containerClassName="checkout-thank-you__domains"
+			sections={ [
+				{
+					sectionKey: 'domain_mapping_whats_next',
+					sectionTitle: translate( 'What’s next?' ),
+					nextSteps: [
+						{
+							stepKey: 'domain_mapping_whats_next_plugin_setup',
+							stepTitle: translate( 'Connect your domain' ),
+							stepDescription: translate(
+								'Setup your connection by following our suggested setup.'
+							),
+							stepCta: (
+								<Button
+									href={ '#' }
+									className={ 'domain-mapping-thank-you__button' }
+									primary
+									busy={ false }
+									disabled={ false }
+								>
+									{ translate( 'Go to setup' ) }
+								</Button>
+							),
+						},
+						{
+							stepKey: 'domain_mapping_whats_next_view_posts',
+							stepTitle: translate( 'Get a profesional email' ),
+							stepDescription: translate(
+								'Add a custom email address to send and receive emails from %(domain)s today.',
+								{
+									args: {
+										domain,
+									},
+								}
+							),
+							stepCta: (
+								<Button
+									href={ '#' }
+									className={ 'domain-mapping-thank-you__button' }
+									busy={ false }
+									disabled={ false }
+								>
+									{ translate( 'Get email' ) }
+								</Button>
+							),
+						},
+					],
+				},
+			] }
+			showSupportSection={ true }
+			customSupportSection={ {
+				description: 'Have questions? Our Happiness Engineers are here if you need help.',
+				links: [
+					{
+						href: CALYPSO_CONTACT,
+						title: 'Ask a question',
+					},
+					{
+						href: SUPPORT_ROOT,
+						title: 'View support documentation',
+					},
+				],
+			} }
```

Comment:
Two questions here:
1. These look like the defaults - do we need to specify them?
2. If you do continue to pass them in, please translate the `title` and `description` strings.
```suggestion
			customSupportSection={ {
				description: translate( 'Have questions? Our Happiness Engineers are here if you need help.' ),
				links: [
					{
						href: CALYPSO_CONTACT,
						title: translate( 'Ask a question' ),
					},
					{
						href: SUPPORT_ROOT,
						title: translate( 'View support documentation' ),
					},
				],
			} }
```

Reply:
Forgot to call `translate` on them. Thanks!

Reply:
These are still missing `translate` calls and still duplicate the default values.

Might it be simpler to just remove this prop definition from here altogether?
```suggestion
```